com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -106,6 +106,12 @@ void InternalLoadResource(VisualEffectResource resource)
             graphView.controller = VFXViewController.GetController(resource, true);
             graphView.UpdateGlobalSelection();
             graphView.FrameNewController();
+            graphView.UpdateIsSubgraph();
```

Comment:
This piece of change is to support the new button in the toolbar to go back to main graph when a subgraph is opened